src/components/Cart/Cart.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect } from "react";
+import { CartProducts } from "../../context/CartProvider";
+
+export function Cart() {
+	const { products, removeFromCart, total } =
+		CartProducts();
+
+	useEffect(() => {});
```

Comment:
Empty useEffect? Not sure if you intend to use it later

Reply:
I'll will change it